scenes/game_elements/props/door/door.tscn

```diff
@@ -0,0 +1,47 @@
+[gd_scene load_steps=6 format=3 uid="uid://lgu7aeqa7o3r"]
+
+[ext_resource type="Script" uid="uid://hop028ljbst7" path="res://scenes/game_elements/props/door/components/door.gd" id="1_t3aaa"]
+[ext_resource type="Texture2D" uid="uid://boeaxk35yjpwl" path="res://scenes/game_elements/props/door/components/door_closed.png" id="2_6x58x"]
+[ext_resource type="Texture2D" uid="uid://efo1k6ymgpuv" path="res://scenes/game_elements/props/door/components/door_opened.png" id="3_wl1pt"]
+
+[sub_resource type="RectangleShape2D" id="RectangleShape2D_t3aaa"]
+size = Vector2(57.5, 25)
+
+[sub_resource type="RectangleShape2D" id="RectangleShape2D_6x58x"]
+size = Vector2(5, 24)
+
+[node name="Door" type="Node2D"]
+scale = Vector2(1.1, 1)
```

Comment:
Is this scale intentional? It gives a warning for the root node when opening the Door scene in the editor: "The root node of a scene is recommended to not be transformed (...)"

```suggestion
```